samples/distributed-tracing/v2/DistributedTracingSample/DistributedTracingSample/DurableFunctionsPatterns.cs

```diff
@@ -0,0 +1,123 @@
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Extensions.Logging;
+
+namespace AddingConfigDistTracingTesting
+{
+    public static class DurableFunctionsPatterns
+    {
+        [FunctionName("FunctionChaining")]
+        public static async Task<List<string>> FunctionChaining(
+            [OrchestrationTrigger] IDurableOrchestrationContext context)
+        {
+            var outputs = new List<string>();
+
+            outputs.Add(await context.CallActivityAsync<string>(nameof(SayHelloActivity), "Tokyo"));
+            outputs.Add(await context.CallActivityAsync<string>(nameof(SayHelloActivity), "Seattle"));
+            outputs.Add(await context.CallActivityAsync<string>(nameof(SayHelloActivity), "London"));
+
+            return outputs;
+        }
+
+        [FunctionName("FanOutFanIn")]
+        public static async Task<string[]> FanOutFanIn(
+            [OrchestrationTrigger] IDurableOrchestrationContext context)
+        {
+            var tasks = new Task<string>[3];
+
+            tasks[0] = context.CallActivityAsync<string>(nameof(SayHelloActivity), "Tokyo");
+            tasks[1] = context.CallActivityAsync<string>(nameof(SayHelloActivity), "Seattle");
+            tasks[2] = context.CallActivityAsync<string>(nameof(SayHelloActivity), "London");
+
+            return await Task.WhenAll(tasks);
+        }
```

Comment:
nit: why not just `.Add()` the tasks?

```suggestion
        [FunctionName("FanOutFanIn")]
        public static async Task<string[]> FanOutFanIn(
            [OrchestrationTrigger] IDurableOrchestrationContext context)
        {
            var tasks = new List<Task<string>>();

            tasks.Add(context.CallActivityAsync<string>(nameof(SayHelloActivity), "Tokyo"));
            tasks.Add(context.CallActivityAsync<string>(nameof(SayHelloActivity), "Seattle"));
            tasks.Add(context.CallActivityAsync<string>(nameof(SayHelloActivity), "London"));

            return await Task.WhenAll(tasks);
        }
```